revscoring/datasources/meta/tests/test_vectorizers.py

```diff
@@ -0,0 +1,24 @@
+from unittest.mock import patch
+import numpy as np
+
+from .. import vectorizers
+from revscoring.datasources import revision_oriented as ro
+from revscoring.dependencies import solve
+from revscoring.features import wikitext
+
+test_vectors = {'a': np.ones(200),
```

Comment:
This is an order of magnitude slower than using the raw python. 

```
>>> start = time.time(); sum(sum([0]*300) for i in range(10000)); time.time() - start
0
0.09108448028564453
>>> start = time.time(); sum(sum(np.zeros(300)) for i in range(10000)); time.time() - start
0.0
0.35784435272216797
```

It would be better to use [1] * 200.   I don't mean to bring this up in a test (who cares), but rather to suggest that the whole of the submitted code should use the [0] * NUM pattern instead.